resources/scripts/download_latest_libs.py

```diff
@@ -168,16 +178,18 @@ def retrieve_libraries_for_specific_version(
         version (str): Version number of the latest bundles.
         output_path (Path): Where to download the libs.
     """
-    logging.info("Attempting to retrieve Connext native libraries ...")
-    url = f"{storage_url}/{storage_path}/{version}"
+    s3 = boto3.client("s3")
 
     for arch, arch_data in data["architectures"].items():
         logging.info(f" - Downloading minimal package for {arch_data['name']} ...")
-
-        response = urllib.request.urlopen(
-            f"{url}/rti_connext_dds-{version}-min-{arch_data['name']}.zip"
+        remote_file_path = PurePosixPath(
+            storage_path,
+            version,
+            f"rti_connext_dds-{version}-min-{arch_data['name']}.zip",
```

Comment:
@lulivi take into consideration what I have to use the minimal package instead of the LM because it does not include the connector library.